### PR TITLE
Removes the muteness disability from the loadout

### PR DESCRIPTION
**Pull request by sheeplets**
Oh boy here I go kicking a hornet's nest.

Despite how rare the condition is in reality mutes have always been weirdly prominent on Aurora for absolutely no reason that I can see besides the fact that the disability is so easily available. People see it and flock to it because it's quirky and offers the illusion of character depth, simple as.

This PR removes the option from character creation while leaving it in-game as a trauma while hopefully killing a few tropes in the process.

Keep in mind that if you play an existing mute character this shouldn't have an impact besides removing your mechanical restriction on speaking, and muteness will still show under your listed disabilities as a leftover. I don't see a reason why this would prevent anyone from roleplaying as they were before.

### CODE REVIEW

**Comment by sheeplets** [comment]
!review

**Review by Doxxmedearly — APPROVED** [review]

**Comment by Greenjoe12345** [comment]
I kinda say this is a bad idea cause of situations like: a mute character gets mind controlled or something, the mind-controller then makes them speak, not knowing the char is ICly mute. The disability meanwhile forces the character to be mute.
that's just one situation on it anyway.

File: html/changelogs/sheepets - RemovesMuteness.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+    - rscdel: "Removed the muteness disablilty from the loadout."
```
Comment (Geevies) [inline html/changelogs/sheepets - RemovesMuteness.yml:41]:
```suggestion
    - rscdel: "Removed the muteness disabililty from the loadout."
```

**Review by Geevies — APPROVED** [review]
dumbass lmao

File: html/changelogs/sheepets - RemovesMuteness.yml (hunk 1 above)
Comment (sheeplets) [reply html/changelogs/sheepets - RemovesMuteness.yml:41]: 2816_DioDefaultDance

**Pull request merged**